backend/projects/models.py

```diff
@@ -57,6 +58,12 @@ class Meta:
         unique_together = ("project", "user")
 
 
+class MetricsPreferences(BaseModel):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    metrics = ArrayField(models.CharField(max_length=100))
```

Comment:
Are you sure that this length is enough ? What is the maximal length with all the metrics selected ?

Reply:
This is the char length for only one metric. `ArrayField(models.CharField(max_length=100))` only tells that the values stored in the array are string with maximum 100 char :)

Reply:
Ah ok great !